pkg/apk/apk/index.go

```diff
@@ -408,3 +452,19 @@ func WithIndexAuthenticator(a auth.Authenticator) IndexOption {
 		o.auth = a
 	}
 }
+
+func redact(in string) string {
+	asURL, err := url.Parse(in)
+	if err != nil {
+		// Attempt to parse non-https elements into URI's so they are translated into
+		// file:// URLs allowing them to parse into a url.URL{}
+		asURL, err := url.Parse(string(uri.New(in)))
+		if err != nil {
+			return in
+		}
+
+		return asURL.Redacted()
```

Comment:
Do we ever need to redact file paths? 🤔

Reply:
I kinda gave up on this one and went with "keep the same behavior but make it a little less janky to read" and this let me just handle them all the same.